File: VB/SLGridExample/SelectionStateToVisibilityConverter.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Windows.Data
Imports DevExpress.Xpf.Grid
Imports System.Windows

Namespace SLGridExample
	Public Class SelectionStateToVisibilityConverter
		Implements IValueConverter

		Public Function Convert(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements IValueConverter.Convert
			Return If(CType(value, SelectionState) = SelectionState.None, Visibility.Visible, Visibility.Collapsed)
		End Function

		Public Function ConvertBack(ByVal value As Object, ByVal targetType As Type, ByVal parameter As Object, ByVal culture As System.Globalization.CultureInfo) As Object Implements IValueConverter.ConvertBack
			Throw New NotImplementedException()
		End Function
	End Class
End Namespace

File: CS/SLGridExample/SelectionStateToVisibilityConverter.cs
using System;
using System.Windows.Data;
using DevExpress.Xpf.Grid;
using System.Windows;

namespace SLGridExample {
    public class SelectionStateToVisibilityConverter : IValueConverter {

        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture) {
            return (SelectionState)value == SelectionState.None ? Visibility.Visible : Visibility.Collapsed;
        }

        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture) {
            throw new NotImplementedException();
        }
    }
}
